CoreWelcomeView/src/au/gov/asd/tac/constellation/views/welcome/WelcomeViewPane.java

```diff
@@ -205,6 +203,16 @@ public WelcomeViewPane() {
                     imageView.setFitHeight(145);
                     imageView.setFitWidth(145);
                     recentGraphButtons[i].setGraphic(imageView);
+                } else {
+                    if (i < fileNames.size()) {
```

Comment:
can merge into `else if` since there is no other code in the else block outside the nested if block